bundle/deepracer_simulation_environment/lib/deepracer_simulation_environment/download_params_and_roslaunch_agent.py

```diff
@@ -45,6 +48,28 @@ def main():
         s3_yaml_name = sys.argv[4]
         launch_name = sys.argv[5]
 
+        LOG.info('S3 region {}'.format(s3_region))
+         # create boto3 session/client and download yaml/json file
+         session = boto3.session.Session()
++        ec2_client = session.client('ec2', s3_region)
++        LOG.info('Checking internet connection...')
++        response = ec2_client.describe_vpcs()
```

Comment:
I think we will need to remove or bypass this new code for running in non-AWS environments.

Reply:
I have now changed it (still needs to be tested) that has one way of working in Local (S3_ENDPOINT_URL defined) and one in AWS.

Reply:
Looks good to me. Merge at your leisure :)